content/blog/2024-06-18-why-you-should-build-on-ethereum-classic-etc-is-proof-of-work/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Why You Should Build on Ethereum Classic: ETC Is Proof of Work"
+date: 2024-06-18
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["education"]
+linkImage: ./banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/5UTFslKruSI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+![](./banner.png)
+
+This post is the first of a series that will explain to dapp developers why they should build on Ethereum Classic (ETC).
+
+The titles of the series will start with the prefix “Why You Should Build on Ethereum Classic” and followed by the highlighted feature of that post.
+
+The 9 highlighted features in this series will be:
+
+- ETC Is Proof of Work
+- ETC is Hard Money
+- ETC Is Programmable
+- ETC Is Fully Replicated
+- ETC Is Composable
+- ETC Is the Largest POW Smart Contracts Blockchain in the World
+- Applications in ETC are the Most Secure Apps in the World
+- ETC Is Code Is Law
+- ETC Is Bitcoin but With Ethereum Technology
+
+In this post we will explain why developers should build on ETC because it is proof of work (POW).
+
+## The Purpose of Blockchains Is Trust Minimization
+
+Trust minimization is the highest value in true blockchains such as ETC, and should also be for your dapps!
+
+The whole point of creating Bitcoin, the first blockchain, was to reduce the dependency on the banking system and central banks.
+
+Indeed, Satoshi Nakamoto mentioned trust minimization 14 times in the Bitcoin white paper!
+
+Trust minimization means that your users should know that their money and functions of your dapp on ETC will not be subject to the whims of any trusted third party or even the developer team.
+
+Additionally, it is not enough for your dapp to be trust minimized by design if the underlying platform on which it is running is not. 
+
+## Trust Minimization Enables Censorship Resistance
+
+For example, you could actually build your app to work the same way on AWS, Azure, or Google Cloud, but you know that those services are centralized and captured by government and special interests, meaning that your app may be censored whenever they wish.
+
+Trust minimization is the opposite of censorship. 
```

Comment:
hmmm, is it really the opposite?

Reply:
Well, not really. We could say that trust minimization" significantly reduces the risk of censorship.